delivery_multi_destination/views/delivery_carrier_view.xml

```diff
@@ -27,7 +27,7 @@
                       attrs="{'invisible': [('destination_type', '!=', 'multi')]}"
                 >
                     <field name="child_ids"
-                           context="{'show_children_carriers': True}"
+                           context="{'show_children_fields': True}"
```

Comment:
Here a try to set `show_children_carriers` to `True` but seems that the action context prevails.